drivers/gpio/Kconfig.sam

```diff
@@ -1,11 +1,21 @@
 # Atmel SAM GPIO configuration options
 
 # Copyright (c) 2018 Justin Watson
+# Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
 # SPDX-License-Identifier: Apache-2.0
 
 config GPIO_SAM
 	bool "Atmel SAM GPIO (PORT) driver"
 	default y
-	depends on SOC_FAMILY_SAM
+	depends on SOC_FAMILY_SAM && !SOC_SERIES_SAM4L
+	select HAS_DTS_GPIO
```

Comment:
We don't need `select HAS_DTS_GPIO`; that's the default given `CONFIG_GPIO=y`.  Probably it should be removed from Kconfig, but leave that to @galak.

Reply:
Uhmm, #24085 removed. Thanks @pabigot !